src/paymentservice/charge.js

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 const cardValidator = require('simple-card-validator');
-const uuid = require('uuid');
+const { v4: uuidv4 } = require('uuid');
```

Comment:
I just copied the official `uuid` npm package docs: https://www.npmjs.com/package/uuid.